conda-forge.yml

```diff
@@ -1,5 +1,9 @@
+build_platform:
+  osx_arm64: osx_64
+conda_forge_output_validation: true
```

Comment:
This key appears to be added twice (please see below), which looks like is causing the bot issues ( https://github.com/regro/cf-scripts/issues/1523 )

Reply:
Good find.